src/shared/modules/dbMeta/dbMetaEpics.ts

```diff
@@ -400,23 +410,25 @@ export const dbCountEpic = (some$: any, store: any) =>
     .throttle(() => some$.ofType(DB_META_COUNT_DONE))
     .mergeMap(() =>
       Rx.Observable.fromPromise<void>(
-        new Promise(async resolve => {
+        (async () => {
```

Comment:
This change was becuase we're technically creating a promise within a promise with the `async` marker here.